src/views/Release/Active.js

```diff
@@ -10,11 +10,13 @@ import { useWallet } from 'store/wallet';
 
 import * as need from 'lib/need';
 import usePermissionsForPoint from 'lib/usePermissionsForPoint';
+import { useSyncOwnedPoints } from 'lib/useSyncPoints';
 import { buildKeyType } from 'lib/point';
 
 function ActiveViewRow({ point, goPoint }) {
   const { wallet } = useWallet();
   const address = need.addressFromWallet(wallet);
+  useSyncOwnedPoints([point]);
```

Comment:
Does the result of this not need to be assigned for something? If we're not using it, shouldn't we remove it?

Reply:
It simply ensures that `point` is synced. It doesn't need to be assigned.